static/js/drafts.js

```diff
@@ -151,7 +151,9 @@ exports.restore_draft = function (draft_id) {
         }
     }
 
-    overlays.close_overlay("drafts");
+    if (!dont_close_overlay) {
```

Comment:
This double negative makes the code kind of difficult to read? If we changed `dont_close_overlay` to `close_overlay` and just do `if (close_overlay) { // close stuff }` makes it a lot easier to read.

Reply:
@lonerz OK, I've changed it to `close_overlay`. I also had to add a condition for if `close_overlay === undefined`, so that other functions calling `restore_draft` wouldn't have to be rewritten. Thanks!